server/models/remote_provider.go

```diff
@@ -3400,3 +3400,43 @@ func (l *RemoteProvider) SetKubeClient(client *mesherykube.Client) {
 func (l *RemoteProvider) GetKubeClient() *mesherykube.Client {
 	return l.KubeClient
 }
+
+// SaveCredential - to save a creadential for an integration
+// TODO
```

Comment:
@Azanul 

Why is this remote_provider support left out? Are you about to add this?

Reply:
@theBeginner86 

There you go